src/components/TranslateMarkdown.tsx

```diff
@@ -11,15 +11,15 @@ interface Props {
 export const TranslateMarkdown = ({ source }: Props) => {
   return (
     <Markdown
-      escapeHtml={true}
```

Comment:
Why are these other props removed?

Reply:
The api has changed. Now html is escaped by default.